ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/RejectReason.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.divorce.ccd.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.ccd.sdk.types.CCD;
+import uk.gov.hmcts.ccd.sdk.types.ComplexType;
+import uk.gov.hmcts.reform.divorce.ccd.model.enums.RejectReasonList;
+
+import static uk.gov.hmcts.ccd.sdk.types.FieldType.FixedList;
+import static uk.gov.hmcts.ccd.sdk.types.FieldType.TextArea;
+
+@Data
+@Builder
+@AllArgsConstructor
+@ComplexType(generate = true)
+public class RejectReason {
+
+    @JsonProperty("RejectReasonType")
+    @CCD(
+        label = "Reject reason",
+        type = FixedList,
```

Comment:
You should only need to provide label here, type and typeParameter should be inferred automatically.